### PR TITLE
correct module-enable-note for capsules and non-satellite builds

### PR DESCRIPTION
**Pull request by evgeni**
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.5/Katello 4.7 (planned Satellite 6.13)
* [x] Foreman 3.4/Katello 4.6
* [x] Foreman 3.3/Katello 4.5 (Satellite 6.12)
* [ ] Foreman 3.2/Katello 4.4
* [ ] Foreman 3.1/Katello 4.3 (Satellite 6.11, orcharhino 6.1+)
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.3 or older.

### CODE REVIEW

File: guides/common/modules/snip_module-enable-note.adoc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Enablement of the module `{dnf-module}` warns about a conflict with `postgresql:10` and `ruby:2.5` as these modules are set to the default module versions on {EL} 8.
```
Comment (ekohl) [inline guides/common/modules/snip_module-enable-note.adoc:1]: Should this file be named `dnf-module-enable`? Module is used so often that I think it should be qualified.

File: guides/common/modules/proc_configuring-repositories-proxy.adoc [hunk 2]
```diff
@@ -31,7 +31,8 @@ Use this procedure to enable the repositories that are required to install {Prod
 +
 [NOTE]
 ====
-include::snip_satellite-el8-enable-note.adoc[]
+:dnf-module: satellite-capsule:el8
```
Comment (ekohl) [inline guides/common/modules/proc_configuring-repositories-proxy.adoc:34]: Perhaps this should be set a bit higher and also used on line 29 so you know they're connected?

**Review by ekohl — COMMENTED** [review]
I'm sure you also debated this, but the `dnf-module` vs `dnf-modules` is a bit vague. Yet I don't know what a better solution would be.

File: guides/common/modules/snip_module-enable-note.adoc (hunk 1 above)
Comment (evgeni) [reply guides/common/modules/snip_module-enable-note.adoc]: yes

**Comment by evgeni** [comment]
> I'm sure you also debated this, but the `dnf-module` vs `dnf-modules` is a bit vague. Yet I don't know what a better solution would be.

Yeah, I didn't see a good way out of this

**Review by ekohl — APPROVED** [review]
Perhaps @maximiliankolb has a good idea here, but I think this is overall an improvement.

File: guides/common/attributes-foreman-el.adoc [hunk 3]
```diff
@@ -1,2 +1,3 @@
 // Overrides for foreman-el build
-:dnf-modules: foreman:el8
+:dnf-module: foreman:el8
```
Comment (maximiliankolb) [inline guides/common/attributes-foreman-el.adoc:2]: I thought it'd be possible to use `{project-context}` here, but it's missing for katello: https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-katello.adoc

I am unsure if this is an issue to begin with.

File: guides/common/attributes-katello.adoc [hunk 4]
```diff
@@ -4,4 +4,5 @@
 :installer-scenario: foreman-installer --scenario katello
 :installer-log-file: /var/log/foreman-installer/katello.log
 :smartproxy_port: 9090
-:dnf-modules: katello:el8 pulpcore:el8
+:dnf-module: katello:el8
```
Comment (maximiliankolb) [inline guides/common/attributes-katello.adoc:7]: Naming suggestion: `main-dnf-module`? I am unsure if this makes things easier. Downside: sort in alphabetical order and the attributes are no longer side by side.

IMHO it's fine as is.

**Review by maximiliankolb — APPROVED** [review]
One small comment/question but it's outside of your PR.

LGTM!

**Comment by evgeni** [comment]
Thanks, I'll keep it as is.

**Pull request merged**